packages/json/tests.pony

```diff
@@ -532,6 +572,32 @@ class iso _TestPrintObject is UnitTest
     // We don't test with more fields in the object because we don't know what
     // order they will be printed in
 
+class iso _TestNoPrettyPrintObject is UnitTest
+  """
+  Test Json none-pretty printing of objects.
+  """
+  fun name(): String => "JSON/nopprint.object"
+
+  fun apply(h: TestHelper) =>
+    let doc: JsonDoc = JsonDoc
+    let obj: JsonObject = JsonObject
+
+    doc.data = obj
+    h.assert_eq[String]("{}", doc.string(false))
+
+    obj.data.clear()
+    obj.data("foo") = true
+    h.assert_eq[String]("{\"foo\": true}", doc.string(false))
+
+    obj.data.clear()
+    obj.data("a") = true
+    obj.data("b") = I64(3)
+    let s = doc.string(false)
+    h.assert_true((s == "{\"a\": true, \"b\": 3}") or
+      (s == "{\"b\": false, \"a\": true}"))
```

Comment:
This clause wouldn't ever be true, right?  It should be `"b": 3`, not `"b": false`.